shell/components/CodeMirror.vue

```diff
@@ -103,6 +104,13 @@ export default {
     },
 
     isCodeMirrorContainerFocused() {
+      // this is a gimmick so that vue only looks at this.$refs in this computed after it's mounted
+      // we do this because this.$refs is not reactive and is "undefined" at the time of the computed's
+      // first evaluation. Addresses https://github.com/rancher/dashboard/issues/13506
+      if (!this.isMounted) {
+        return false;
+      }
```

Comment:
optional: Alternatively, we could use a reactive variable (e.g. `codeMirrorRef`) to store the ref during the mounted hook. This might be a little more clear and require less explanation about what is happening; it also might look less like a gimmick.

Reply:
You're right @rak-phillip , that's much neater and is reactive 🙇